include/onnxruntime/core/optimizer/graph_transformer.h

```diff
@@ -36,7 +42,7 @@ class GraphTransformer {
   @param[out] modified Set to true if the Graph was modified.
   @returns Status with success or error information.
```

Comment:
Document 'providers' input, and maybe move it in front of 'modified' given it's an input param.